test/WebJobs.Script.Tests/Binding/ActionResults/RawScriptResultTests.cs

```diff
@@ -61,13 +61,14 @@ public async Task AddsHttpCookies()
                 {
                     new Tuple<string, string, CookieOptions>("firstCookie", "cookieValue", new CookieOptions()
                     {
-                        SameSite = SameSiteMode.None
+                        SameSite = SameSiteMode.Lax
                     }),
                     new Tuple<string, string, CookieOptions>("secondCookie", "cookieValue2", new CookieOptions()
                     {
                         Path = "/",
                         HttpOnly = true,
-                        MaxAge = TimeSpan.FromSeconds(20)
+                        MaxAge = TimeSpan.FromSeconds(20),
+                        SameSite = (SameSiteMode)(-1)
```

Comment:
We should be able to use the new enum value here instead (you may need to add an explicit reference to the package here)

@brettsam, we may need to go over the projects to make sure we're building against the right versions.

Reply:
We were trying to not update Script to use anything from 2.0, if I recall. Only WebHost moved forward.

Reply:
Was there a good reason for that? It will lead to behavior like this where we're building against an incorrect API